### ISSUE TITLE
Negative Decimal problem

### ISSUE THREAD

**Issue by gvaughn** [danielberkompas/number#28]
```
iex(15)> Number.Currency.number_to_currency(Decimal.new(-100.01))
"$100.01"
```

Looks like the `number < 0` guard clauses don't work with Decimal because:
```
iex(16)> Decimal.new(-100.01) < 0
false
```

**Comment by danielberkompas** [comment]
@gvaughn I've created a PR to fix this: #29

**Comment by gvaughn** [comment]
Thanks! I was just about to fork and create a PR, but you're too quick for me. I wasn't familiar with decimal_arithmetic before -- very interesting
